tests/hooks/test_sqlite_hook.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+import unittest
+from unittest import mock
+from unittest.mock import patch
+
+from airflow.hooks.sqlite_hook import SqliteHook
+from airflow.models import Connection
+
+
+class TestSqliteHookConn(unittest.TestCase):
+
+    def setUp(self):
+        super().setUp()
```

Comment:
I think this is not required.

Reply:
I tried to write tests for this hook like as other hook and in other hooks this is exists.

Reply:
Mind remove it and check whether if it is necessary. It is possible that other hook tests added the method by mistake.

Reply:
Okay. I will check it.

Reply:
Done